src/utils/axiosInterceptors.ts

```diff
@@ -82,6 +94,14 @@ const setCacheResponse = async (response: any): Promise<any> => {
     }
 
     let cacheKey;
+
+    const expires = new Date();
+    expires.setSeconds(expires.getSeconds() + EXPIRES_IN_SECONDS);
```

Comment:
Cool, except maybe the expiry time should be a parameter when making requests? For some requests it might make sense to expire in a month, and for some (images) half an hour would be better... It depends on the application too I guess.

If you think this is outside of scope, fftr and we can deal with it later.

Reply:
I think we can add this in a different PR. So will resolve for now